client/test/formatting/after_formatting.ion

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Could we add some lists to the test as well as some s-expressions? 
Does the ion-js formatter deal with long lines? Do they get wrapped or splt?

Reply:
Sure, I will add test for list and sexps.